### DISCUSSION TITLE
Use Camel-karavan with OCI

### DISCUSSION THREAD

**Discussion by Shahid-Roofi-Khan** [apache/camel-karavan#613] [Q&A] [answered]
Hi Team,

My team want to use Camel-karavan with Oracle cloud kubernetes engine. Any clues to steps? if so please share to save us time.

**Comment by mgubaidullin** [comment, marked as answer]
I have never tried OCI. If it is akubernetes than I believe aws example could help https://github.com/apache/camel-karavan/tree/main/karavan-cloud/AWS
